docs/api.md

```diff
@@ -0,0 +1,49 @@
+# Renovate API
+
+Renovate CE/EE exposes a REST API that can be used to interact programmatically with Renovate.
+
+## Enabling and Authentication
+
+The API can be enabled by setting the `MEND_RNV_ADMIN_API_ENABLED` environment variable to `true`.
+You must also configure an API secret by setting the `MEND_RNV_SERVER_API_SECRET` variable.
+
+Authentication is done via HTTP Auth, using the API secret as the password.
+For example if the secret is `renovateapi` then you would authenticate with:
+
+```
+  Authorization: renovateapi
+```
+
+## Endpoints
+
+### Health
+
+`GET /health`
+
+Returns 200 if API is healthy.
+
+### Queue status
+
+`GET /api/job/queue`
+
+Returns the current status of the job queue, including number of pending jobs.
+
+`GET /api/task/queue`
+
+Returns the current status of the task queue, including number of pending tasks.
+Generally speaking, tasks are internal implementation details (such as sync'ing) so this API is exposed primarily for the purpose of troubleshooting if something is going wrong.
```

Comment:
```suggestion
Generally speaking, tasks are internal implementation details, such as syncing.
As end-user you usually do not need to worry about tasks.
This API is exposed primarily to help you troubleshoot if something is going wrong.
```

We should split up the long sentence into two (or three) parts:

- Tasks is internal implementation detail
- API is for troubleshooting
- _Optional:_ explain that you usually won't need to worry about tasks at all